src/contributing/contributing.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Contributing to the dxw Accessibility Manual
+navigation_order: 0
+---
+
+The dxw Accessibility Manual is public and anyone, inside or outside of dxw, can suggest changes to it. This means you!
+
+We manage changes via GitHub ‘Pull Requests’. This is a technical term for requesting a specific change is made to a repository of text. Often this text is code, but in this case it’s the contents of this Playbook. It allows everyone to see exactly what changes are being proposed, and for discussion and changes to be made before it is ‘merged’, and those changes to be recorded for the future.
+
+If you’re experienced in doing this, then all you need to know is that this repository is available at https://github.com/dxw/playbook
+
+If creating a pull request is new to you, then don’t worry, here is a guide to use Netlify CMS to create one for you.
+
+## What you will need
+
+You will need a GitHub account. If you don’t already have one, then sign up here. You do not need to be added to the dxw GitHub organisation to be able to suggest changes.
+
+## Suggesting a change
+
+1. Open the page for editing
+
+For an existing page:
+
+Navigate to the page you want to edit
+Click on the yellow ‘Edit this page’ link
+For an new page:
+
+Navigate to the most relevant ‘parent’ for the new page
+Click on the yellow ‘Add a new page here’ link
+Alternatively, if you prefer to use the admin backend directly:
```

Comment:
These two blocks become two paragraphs. You'll want some list or number list markup here and in some other places I think

You can preview what this looks like here: https://deploy-preview-133--startling-pasca-a7e8b4.netlify.app/contributing/contributing

Reply:
Thank you, I got stuck with the double sets of numbering in the Playbook so will rewrite this section